jemallocator-global/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "jemallocator-global"
+version = "0.2.0"
+authors = ["gnzlbg <gonzalobg88@gmail.com>"]
+edition = "2015"
+license = "MIT/Apache-2.0"
+readme = "README.md"
+keywords = ["allocator"]
+repository = "https://github.com/alexcrichton/jemallocator"
+homepage = "https://github.com/alexcrichton/jemallocator"
+documentation = "https://docs.rs/jemallocator"
+description = """
+Sets jemalloc as the #[global_allocator]
+"""
+
+[dependencies]
+jemallocator = { optional = true, path = "..", version = "0.2.0" }
+
+[target.'cfg(all(windows, target_env = "msvc"))'.dependencies]
+jemallocator = { optional = false, path = "..", version = "0.2.0" }
```

Comment:
IIRC, the feature associated with a non-optional dependency still needs to be explicitly enabled, so I think this setup will no-op on all platforms unless you enable the force or jemallocator features explicitly.

Reply:
I was unsure about that, but I tried commenting/uncommenting these w/o `--features=force` and it appeared to work properly :/

Reply:
So I think I've resolved this!